check/src/typecheck.rs

```diff
@@ -1670,6 +1674,7 @@ impl<'a> Typecheck<'a> {
                         })
                     }
                     Type::Forall(ref params, ref typ, _) => {
+                        println!("{:?}", self.type_variables);
```

Comment:
Debugging stuff left in? 😆

Reply:
Yah, it is in the last WIP commit :). Making sure it is backed up since I have had some computer issues recently.